frontend/prebuild/src/environments/environment.ts

```diff
@@ -4,5 +4,10 @@
 // The list of which env maps to which file can be found in `.angular-cli.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  API_URL_AUTH: 'http://localhost:8082',
+  API_URL_GAME_ROUND: 'http://localhost:8080/round',
+  API_URL_GAME_WS: 'ws://localhost:8080/round/ws',
+  API_URL_PLAYERS: 'http://localhost:8081/player',
+  API_URL_RANKS: 'http://localhost:8081/rank'
```

Comment:
Right now, if the project gets built/run with prod mode these won't be available. We might consider adding these variables to `environment.prod.ts` with at least placeholder values (if not these same values) so that we aren't wondering why it's not working in prod mode.

Reply:
Actually if we do `ng build --env=prod` right now, we get compile errors for the properties missing, which I think is more desireable than putting in fake values that will allow the app to make it all the way to deployment until things fail.

The error message is pretty straightforward:
> ERROR in src/app/game/game.service.ts(19,40): error TS2339: Property 'API_URL_GAME_WS' does not exist on type '{ production: boolean; }'.